### PR TITLE
use a crisper svg logo

### PR DESCRIPTION
**Pull request by KTibow**
svgs are vector files. they can get small with optimization, smaller with manual tweaking, and even smaller with gzip compression, while having better resolution at the same time.

the logo on banglejs.com/apps/ looks blurry, and it gets worse on high density displays. this pr fixes it by moving it to an svg.

(i plan to make some more prs after this one that make the ui better in some other ways)

### CODE REVIEW

File: android.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <body>
     <header class="navbar-primary navbar">
       <section class="navbar-section" >
-        <a href="https://banglejs.com" target="_blank" class="navbar-brand mr-2" ><img src="img/banglejs-logo-sml.png" alt="Bangle.js">
+        <a href="https://banglejs.com" target="_blank" class="navbar-brand mr-2"><img src="img/banglejs-logo-small.svg" alt="Bangle.js">
```
Comment (bobrippling) [inline android.html:25]: Do we want to delete `banglejs-logo-sml.png`?

**Review by bobrippling — COMMENTED** [review]
Nice idea - cc'ing @gfwilliams since it's a more user-facing change

File: android.html (hunk 1 above)
Comment (gfwilliams) [reply android.html:25]: Maybe don't bother? It's not like it's a big file...

**Comment by gfwilliams** [comment]
Looks good - thanks!

File: android.html (hunk 1 above)
Comment (KTibow) [reply android.html:25]: I feel like if it were to be deleted it would be more a matter of semantics

Comment (bobrippling) [reply android.html:25]: Yeah my thought was that someone might change one without the other

**Pull request merged**